frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -10,6 +10,7 @@ type Properties = {
     size?: 'md' | 'lg';
     isDisabled?: boolean;
     sx?: SystemStyleObject;
```

Comment:
Can you pass class names instead of sx?

Reply:
I think we need to create a separate ticket for this because we used buttons with sx everywhere in the code before. And I don't want to change the button values from sx to classname everywhere in this PR.